crates/proxyutil/src/cloudauth.rs

```diff
@@ -63,12 +65,12 @@ impl ServiceProtocol {
 /// Params used for cloud authentication.
 #[derive(Debug, Clone)]
 pub struct AuthParams<'a> {
-    pub user: &'a str,
-    pub password: &'a str,
-    pub db_name: &'a str,
+    pub user: Cow<'a, str>,
+    pub password: Cow<'a, str>,
+    pub db_name: Cow<'a, str>,
```

Comment:
Why are we using a `Cow` here? Just curious, unable to infer this by glancing over the code.

Reply:
The original struct was all `&'a str`. Now with the auth implementation for flight, we need to operate on some owned values. So instead of unnecessarily converting everything to an owned value, just used Cow so we can use owned & borrowed values.

Reply:
Oh yeah, that makes sense